src/domain/permissions.js

```diff
@@ -100,34 +62,15 @@ export function getAllUserPermissions(user) {
 }
 
 export default {
-  ADMIN,
-  EDIT_DOC,
-  VIEW_DOCS,
-  REVIEW_DOC,
-  VERIFY_DOC,
-  PROTECT_ANY_DOC,
-  PROTECT_OWN_DOC,
-  EDIT_FILE,
-  VIEW_FILES,
-  DELETE_OWN_FILES,
-  CREATE_FILE,
-  SEARCH_USERS,
+  CREATE_CONTENT,
+  ACCESS_STORAGE,
+  DELETE_OWN_PRIVATE_CONTENT,
+  PROTECT_OWN_PUBLIC_CONTENT,
+  VIEW_USERS,
+  MANAGE_PUBLIC_CONTENT,
+  MANAGE_PROTECTORS_OF_OWN_PUBLIC_CONTENT,
   MANAGE_USERS,
-  MANAGE_BATCHES,
-  MANAGE_SETTINGS,
-  MANAGE_STORAGE_PLANS,
-  HARD_DELETE_SECTION,
-  SEE_USER_EMAIL,
-  MIGRATE_DATA,
-  ARCHIVE_DOC,
-  RESTORE_DOC_REVISIONS,
-  DELETE_ANY_STORAGE_FILE,
-  OWN_ROOMS,
-  DELETE_FOREIGN_ROOMS,
-  AUTORIZE_ROOMS_RESOURCES,
-  JOIN_ROOMS,
-  CREATE_DOCUMENT_COMMENTS,
-  MANAGE_DOCUMENT_COMMENTS,
-  MANAGE_CONTENT,
-  MANAGE_ACCREDITED_EDITORS
+  MANAGE_SETUP,
+  BATCH_PROCESS_DATA,
+  DELETE_ANY_PRIVATE_CONTENT
 };
```

Comment:
If you're fond of the cleanliness and look at these changes, you feel a little bit like in heaven :heart:

Reply:
Which both of us are found of, yes! It's indeed a breath of fresh air.
Simplicity, cleanliness, straight-forwardness, with so much less risk of misuse and bugs ❤️